kstreams/streams.py

```diff
@@ -93,6 +96,18 @@ async def func_wrapper(func):
         await self.consumer.start()
         self.running = True
 
+        if self.initial_offsets is not None:
+            if isinstance(self.topics, List):
```

Comment:
`self.topics` is always a `list` : https://github.com/kpn/kstreams/blob/master/kstreams/streams.py#L56